meinberlin/apps/polls/exports.py

```diff
@@ -40,3 +42,93 @@ def get_virtual_fields(self, virtual):
     @property
     def raise_exception(self):
         return self.request.user.is_authenticated
+
+
+class PollExportView(export_views.BaseItemExportView):
+
+    def get_queryset(self):
+        creators_vote = poll_models.Vote.objects.filter(
+            choice__question__poll=self.poll).values_list('creator', flat=True)
+        creators_answer = poll_models.Answer.objects.filter(
+            question__poll=self.poll).values_list('creator', flat=True)
+        creator_ids = list(set(creators_vote).union(set(creators_answer)))
+        return User.objects.filter(pk__in=creator_ids)
```

Comment:
@fuzzylogic2000 Because of the User model, we would need to keep this here, right? But all of the rest, I could move into a mixin and add to A4 instead, what do you think?

Reply:
Ah, cool! So we would then just leave an empty view in mB and the rest can go to A4?